adwords/report-builder.js

```diff
@@ -48,7 +48,7 @@ class AdwordsReportBuilder {
      * @param fields {array}
      */
     buildFields(selector, fields) {
-        for (var index in fields) {
+        for (var index = 0; index < fields.length; index++) {
```

Comment:
is it possible for fields be to null when users do not specify the fields? This line would crash if they pass in an undefined value.